lib/mix/tasks/clone.ex

```diff
@@ -0,0 +1,79 @@
+defmodule Mix.Tasks.Umwelt.Clone do
+  @moduledoc "Clones phase modules and code"
+  @shortdoc "The code puller"
+  use Mix.Task
+  require Logger
+
+  @impl Mix.Task
+  def run([phase_id]) do
+    case System.get_env("UMWELT_TOKEN", "no_token") do
+      "no_token" ->
+        """
+        Token not found in env!
+        You can get it on umwelt.dev/auth/profile and do 
+
+          export UMWELT_TOKEN="token"
+
+        or pass it directly in 
+
+          mix clone phase_id "token"
+
+        """
+        |> Logger.warning()
+
+      token ->
+        run([phase_id, token])
+    end
+  end
+
+  @impl Mix.Task
+  def run([phase_id, token]) do
+    Umwelt.Client.Application.start(nil, nil)
```

Comment:
It does not matter much, but it makes sense to [conform the spec](https://hexdocs.pm/elixir/Application.html#c:start/2).
```suggestion
    Umwelt.Client.Application.start(:normal, [])
```

Reply:
Fixed